app/controllers/healthcheck_controller.rb

```diff
@@ -1,6 +1,6 @@
 require 'api_entity'
 
-class HealthcheckController < ActionController::Base
+class HealthcheckController < ApplicationController
```

Comment:
Actually, its worth double checking that this doesn't break anything after you've deployed this

https://dev.trade-tariff.service.gov.uk/healthcheck

Reply:
Nice, yeah there is also some rebasing to do with this branch aswell